From 76cb637dc56f57d1428071cc08903c4bba8a7cc9 Mon Sep 17 00:00:00 2001
From: Nate Drude <nate.d@variscite.com>
Date: Thu, 20 Aug 2020 09:02:53 -0500
Subject: [PATCH] imx8mn: Set IMX_BOOT_UART_BASE for console on uart4

i.MX8MN ATF code sets IMX_BOOT_UART_BASE to UART2.
On Variscite i.MX8MN boards, the console is on UART4.
imx-atf will hang on Variscite boards without this change.

Signed-off-by: Nate Drude <nate.d@variscite.com>
---
 plat/imx/imx8m/imx8mn/include/platform_def.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plat/imx/imx8m/imx8mn/include/platform_def.h b/plat/imx/imx8m/imx8mn/include/platform_def.h
index 967afbbee..669ffe1b1 100644
--- a/plat/imx/imx8m/imx8mn/include/platform_def.h
+++ b/plat/imx/imx8m/imx8mn/include/platform_def.h
@@ -51,7 +51,7 @@
 
 #define HAB_RVT_BASE			U(0x00000900) /* HAB_RVT for i.MX8MM */
 
-#define IMX_BOOT_UART_BASE		U(0x30890000)
+#define IMX_BOOT_UART_BASE		U(0x30a60000)
 #define IMX_BOOT_UART_CLK_IN_HZ		24000000 /* Select 24MHz oscillator */
 
 #define PLAT_CRASH_UART_BASE		IMX_BOOT_UART_BASE
